tests/should-pass/iati-activities/iati-activity/result/indicator/_aggregation-status/attribute-exists.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity> <!-- at least 1 -->
+    <iati-identifier></iati-identifier>  <!-- 1 and only 1-->
+    <reporting-org type="xx" ref="xx"><narrative>Organisation name</narrative></reporting-org> <!-- 1 and only 1-->
+    <title> <!-- 1 and only 1-->
+      <narrative>Xxxxxxx</narrative> <!-- At least 1-->
+    </title>
+    <description>
+      <narrative>Xxxxxxx</narrative> <!-- At least 1-->
+    </description>
+    <participating-org role="xx"></participating-org> <!-- At least 1-->
+    <!--<other-identifier></other-identifier>-->
+    <activity-status code="xx"/> <!-- 1 and only 1-->
+    <activity-date type="xx" iso-date="2013-11-27"/><!-- At least 1 --> <!--Narative allowed-->
+    <activity-date type="xx" iso-date="2013-11-27"><!-- At least 1 --> <!--Narative allowed-->
+      <narrative>Some stuff ere</narrative>
+    </activity-date>
+    <result type="xx">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <indicator measure="xx" aggregation-status="false"><!-- optional aggregation-status attribute included with permitted value -->
```

Comment:
For an optional boolean attribute, should be tests for:

* `false` or `true` (string-based valid value) - **PRESENT**
* `1` or `0` (numeric valid value) - **NOT PRESENT**
* `xx` (invalid value) - **PRESENT**
* something where the attribute isn't there - **ISN'T NOTED IN THIS PR** (might already exist, though isn't clear)

Reply:
The last test is included at the `result` level of test data as `indicator` is a required element for the `result` element.